source/math/Quaternion.ooc

```diff
@@ -33,7 +33,7 @@ Quaternion: cover {
 	// z = this y
 	// w = this z
 
-	inverse ::= This new(this w, -this x, -this y, -this z)
+	inverse ::= This new(this w / (this norm * this norm), -this x / (this norm * this norm), -this y / (this norm * this norm), - this z / (this norm * this norm))
```

Comment:
Alright, well, any depending code will have to be updated, or things will go bad. If what you say is true, then it should be easy. Without committing anything further in `ooc-kean`,

1. Rename `inverse` here to `inverse2` or something (but don't commit)
1. Try to build the project
1. Any `inverse` call on a `Quaternion` will cause an error, so switch for `conjugate`, commit, and make another PR in the other project
1. I will merge the other PR first, and then this one.